pangeo-binder/values.yaml

```diff
@@ -27,6 +29,10 @@ binderhub:
           kubernetes.io/ingress.class: nginx
           kubernetes.io/tls-acme: 'true'
     cull:
+      # don't cull authenticated users
+      users: False
+      namedServerLimitPerUser: 5
+      redirectToServer: false
```

Comment:
These 2 configs (`namedServerLimitPerUser` and `redirectToServer`) have no effect here, they should be under `jupyterhub.hub`

Reply:
Note: there is a bug with `namedServerLimitPerUser` config (https://github.com/jupyterhub/jupyterhub/issues/2758). It is fixed by https://github.com/jupyterhub/jupyterhub/pull/2772 but it is not released yet.